game_patch/multi/server_internal.h

```diff
@@ -34,6 +56,7 @@ struct ServerAdditionalConfig
     VoteConfig vote_restart;
     VoteConfig vote_next;
     VoteConfig vote_previous;
+    NewSpawnLogicConfig random_spawns;
```

Comment:
My suggestion:
```suggestion
    NewSpawnLogicConfig new_spawn_logic;
```

Reply:
This started out as `random_spawns` as a bool and evolved beyond that, but I forgot to go back and change the term. Will make that adjustment.